### DISCUSSION TITLE
default select option

### DISCUSSION THREAD

**Discussion by iateadonut** [livewire/livewire#5410] [1. Help]
If I have a list of states, and I want the default to be "FL", for example, but want to leave it in the middle of an alphabetical listing, how would I do that?

**Comment by joshhanley** [comment]
@iateadonut you would just set the default in your Livewire property in your component that you have `wire:model` on the select, like:

```php
public $selectedState = "FL";
```

Then in your blade view you would just output your states in Alphabetical order and "FL" should be selected.

Hope this helps!
